launchcontainer/tests.py

```diff
@@ -127,11 +127,11 @@ def test_studio_view(self, fragment, render_template):
         # Confirm that the context was set properly on the XBlock instance.
         cls = type(block)
         context = render_template.call_args_list[0][0][1]
-        self.assertEqual(tuple(context['fields']), (
-            (cls.project, 'Foo project', 'string'),
-            (cls.project_friendly, 'Foo Project Friendly Name', 'string'),
-            (cls.project_token, 'Foo token', 'string')
-        ))
+        # self.assertEqual(tuple(context['fields']), (
```

Comment:
This is indeed a complex `assert`, I think it should be broken into multiple `assert`'s instead.